### ISSUE TITLE
Clear read button

### ISSUE THREAD

**Issue by rasocbo** [rasocbo/bublol#38]
There's a really neat feature called "clear read" that clears or hides the posts/comments that the user has supposedly read or scrolled past. It's a feature inspired from the reddit mobile app Boost. 

The current design spec shows it placed as a static button near the top:

![image](https://user-images.githubusercontent.com/68491343/101216900-598fcf00-36a6-11eb-8660-e829981a752c.png)

But I would rather have it always visible, preferably at the top-right corner. 

Current implementation, on the "/posts" page looks like this:

[![](https://thumbs.gfycat.com/GlaringInfantileFly-size_restricted.gif "https://gfycat.com/GlaringInfantileFly")](https://giant.gfycat.com/GlaringInfantileFly.webm)

It currently doesn't have a place on specific "/post/:id" page.

Another thing with this button, is that it's supposed to shrink after a while because I imagine regular users would get familiar and wouldn't want the whole "clear read" text permanently occluding whatever is underneath (since it'll be a sticky button). Current icon (back arrow) could seem ambiguous. Earlier I had a "trash" like icon 🧹. 

- [ ] Does this look OK on the "/posts" page?
- [ ] Where to place it on a specific "/post/:id" page.
- [ ] An icon that can convey meaning when used without text.

**Comment by dragosjurca** [comment]
> * [ ]  Does this look OK on the "/posts" page?

I agree that, if fixed, it is too big and it obfuscates the layout this way.

> * [ ]  An icon that can convey meaning when used without text.

An inversed refresh button might be better in this case, instead of the current clear/back or broom/eraser/delete icon. It has a more natural indication and I believe it to be more intuitive to its meaning. There‘s also the mouse hover over to remind the user, just for extra measure. The tooltip can also include more than what it does now; so for example, it could also say “Refresh feed and clear old content”.

> * [ ]  Where to place it on a specific "/post/:id" page.

Maybe nowhere at all since “/post/:id” is on another level and so this page does not serve the same purpose as “/posts”. As shown in the GIF below, the “Clear read” icon gets replaced with a “Go back [to ~/posts]” button, in its same position underneath the theme button. (For the other “Clear read” button in the comments section, I will come up with a solution in the coming days.)

![Figma](https://user-images.githubusercontent.com/45625917/101525608-35790a00-3994-11eb-870d-ac9c7fac7c09.gif)
